examples/1Intro/4Graphulo/DBsetup.m

```diff
@@ -0,0 +1,32 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% Setup binding to a database.
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+myName = 'mytable_';      % SET LOCAL LABEL TO AVOID COLLISIONS.
+
+[DB,G] = DBsetupLLGrid('txg-graphulotest-02');            % Create binding to database.  Shorthand for:
+% DB = DBserver('llgrid-db-00.llgrid.ll.mit.edu:2181','Accumulo','accumulo','AccumuloUser',password);
```

Comment:
DBserver command doesn't return Graphulo object, should fix this so DBsetupLLGrid isn't requierd